src/main/resources/templates/smallfull/addOrRemoveRptTransactions.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html
+        xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" xmlns:th="http://www.w3.org/1999/xhtml"
+        layout:decorate="~{layouts/baseLayout}">>
+<head>
+    <meta charset="UTF-8">
+    <title id="page-title-heading">
+        Related party transactions
+    </title>
+</head>
+<body>
+<div id="criteria-main-content" layout:fragment="content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+
+            <form id="correspondence-form" class="form" th:action="@{''}" method="post">
+                <div th:replace="fragments/numberedHeading :: numberedHeading (
+                  headingText = 'Related party transactions'
+              )">
+                </div>
+
+                <span id="relatedPartyTransactions-help-text" class="govuk-hint">
+                You can report up to 20 related party transactions.
```

Comment:
Shouldn't this page have all of the input fields as well as the Add another button but with no functionality surrounding them? As the story on JIRA suggests implementing the page but no functionality.

Reply:
Added fields to the page